c_src/build_deps.sh

```diff
@@ -13,8 +13,7 @@ SCRIPT=$SCRIPTPATH/${0##*/}
 BASEDIR=$SCRIPTPATH
 BUILD_CONFIG=$BASEDIR/rocksdb/make_config.mk
 
```

Comment:
Fix the url in the commit comment to reference NOT master BUT the specific commit ID